EventsExpress/ClientApp/src/components/event/filter/parts/organizer/organizer-filter.js

```diff
@@ -0,0 +1,62 @@
+import { FilterExpansionPanel } from '../../expansion-panel/filter-expansion-panel';
+import { change, Field, getFormValues } from 'redux-form';
+import React, { useEffect } from 'react';
+import { Chip } from '@material-ui/core';
+import { useOrganizerFilterStyles } from './organizer-filter-styles';
+import { connect } from 'react-redux';
+import OrganizerAutocomplete from './organizer-autocomplete';
+import { fetchUsers } from '../../../../../actions/events/filter/users-data';
+
+const OrganizerFilter = ({ dispatch, organizers, formValues }) => {
+    const classes = useOrganizerFilterStyles();
+
+    const deleteOrganizer = organizerToDelete => {
+        return () => {
+            dispatch(change(
```

Comment:
Please do the mapDispatchToProps instead of invoking dispatch method directly. It creates a tightly bound dependency.

Reply:
Done